x-pack/plugins/actions/server/config.ts

```diff
@@ -145,15 +147,14 @@ export const configSchema = schema.object({
       max: schema.maybe(schema.number({ min: MIN_QUEUED_MAX, defaultValue: DEFAULT_QUEUED_MAX })),
     })
   ),
-  usage: schema.maybe(
-    schema.object({
-      ca: schema.maybe(
-        schema.object({
-          path: schema.string(),
-        })
-      ),
-    })
-  ),
+  usage: schema.object({
+    url: schema.string({ defaultValue: DEFAULT_USAGE_API_URL }),
```

Comment:
nit: May I suggest that this is `schema.maybe(schema.string())`?

This way, we've got a way to disable the feature by not providing the URL in the config.

WDYT?

Reply:
I am planning to use the `ca` for that purpose.
And this is temporary, there is an issue to create a plugin that provides `ca` and `url` from kibana-controller.